### PR TITLE
Add the hqmf identifier to a statement reference

### PR DESCRIPTION
**Pull request by dczulada**
This is a work in progress for populating the hqmf_identifiers for the populations in a population set.  suggest someone more familar with the code base review/modify accordingly.

relates to https://github.com/projecttacoma/cqm-models/pull/85

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/measure-loader/hqmf_measure_loader.rb [hunk 1]
```diff
@@ -80,21 +84,21 @@ def extract_population_set(population_hqmf_node)
           statement_ref_string = statement_ref.attr('extension')
           case cc.name
           when 'initialPopulationCriteria'
-            ps[:populations][HQMF::PopulationCriteria::IPP] = statement_ref_string
+            ps[:populations][HQMF::PopulationCriteria::IPP] = statement_ref_string + ".#{cc.at_css('id').attr('root')}"
```
Comment (hossenlopp) [inline lib/measure-loader/hqmf_measure_loader.rb]: Why not use a hash to break apart the parsed statement reference and hqmf_id instead of concatenating it on to pull it apart later in `modelize_statement_ref_string`.

Comment (dczulada) [reply lib/measure-loader/hqmf_measure_loader.rb]: I was just using the pattern that already existed.

Comment (dczulada) [reply lib/measure-loader/hqmf_measure_loader.rb]: I made the changes into a Hash.
